index.js

```diff
@@ -17,14 +17,17 @@
 
 const config = require('./config/config');
 config.version = require('./package.json').version;
-const debug = require('debug')('subsituter');
+const debug = require('debug')('substituter');
 const mongoose = require('mongoose');
 const backoff = require('backoff');
 
 // check fs & create dirs if necessary
 const fse = require('fs-extra');
 fse.mkdirsSync(config.fs.base);
 fse.mkdirsSync(config.fs.compendium);
+// const fs = require('fs');
+const yaml = require('js-yaml');
+// const dirTree = require('directory-tree');
 
 const dbURI = config.mongo.location + config.mongo.database;
 mongoose.connect(dbURI);
```

Comment:
Please configure MongoDB to use native promises as you can see in the diffs [here](https://github.com/o2r-project/o2r-contentbutler/commit/675ff2deac23febb31cbfd4ce51e31d2e57b5088#diff-168726dbe96b3ce427e7fedce31bb0bcR28) and [here](https://github.com/o2r-project/o2r-contentbutler/commit/675ff2deac23febb31cbfd4ce51e31d2e57b5088#diff-168726dbe96b3ce427e7fedce31bb0bcR160).

Reply:
done